electron/webpack/webpack.config.renderer.ts

```diff
@@ -0,0 +1,107 @@
+import path from 'path';
+import webpack from 'webpack';
+import 'webpack-dev-server';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+import DotEnv from 'dotenv-webpack';
+import TerserPlugin from 'terser-webpack-plugin';
+
+import paths, { resolveApp } from './paths';
+
+export const rendererConfig: webpack.Configuration = {
+  mode: 'development',
```

Comment:
Should this be an env var?

Reply:
Just defaults to `development` mode but it can be overridden by CLI or `NODE_ENV` variable. See here: https://webpack.js.org/configuration/mode/